game-script.js

```diff
@@ -161,19 +165,20 @@ function start() {
 
 
 
+      // UWAGI AS: nieczytelny if, nie kumam co sie w nim dzieje. męczy samo patrzenie na niego :(
       if ((
         (((leftEdgePplayer <= leftEdgeFruit) && (leftEdgeFruit <= rightEdgePlayer)) ||
           ((leftEdgePplayer <= rightEdgeFruit) && (rightEdgeFruit <= rightEdgePlayer))) &&
         ((bottomEdgeFruit >= topEdgePlayer) && (bottomEdgeFruit <= bottomEdgePlayer)))
-        && fruit.firstTouch == false
+        && fruit.firstTouch == false  // UWAGI AS: "fruit.firstTouch == false" to to samo co "!fruit.firstTouch"
```

Comment:
próbowałam to zapisać bez zmiennych let, na samych wartościach fruit.x itd. i moim zdaniem wtedy wygladał lepiej, ale dostałam uwagę, żeby powatrzające się fragmenty kodu zapisać w zmiennych. Tak zrobiłam. Co do samefo if'a zależy mi, żeby tak działał, a nie wiem jak zapisać go inaczej. Sprawdza, czy lewa bądź prawa krawędz owoca znajduje się pomiędzy krawędziami playera + wysokośc